openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/Widget.js

```diff
@@ -153,7 +153,7 @@ export default {
         }
       }
       // If they pass null for max_params we don't check for a maximum number
-      if (max_num_params && this.parameters[max_num_params] !== undefined) {
+      if (max_num_params !== null && this.parameters.length > max_num_params) {
```

Comment:
both this line and the min_num_params logic is confusing to me.

Are there usecases that checking `parameter.length` don't satisfy?

Reply:
Probably not. The main issue is that we're only using `verifyNumParams` in two places: LabelWidget and ValueWidget. If more widgets were using this routine (and they probably should) then this code would be improved.

Reply:
Created https://github.com/OpenC3/cosmos/issues/1210 to address this